great_expectations/util.py

```diff
@@ -121,6 +116,16 @@ def __delitem__(self, key: str):
         super().__delitem__(key)
 
 
+def get_context(*args, **kwargs) -> AbstractDataContext:
+    # deprecated-v0.17.12
+    warnings.warn(
+        "Importing `get_context` from `great_expectations.util` is deprecated as of v0.17.12 and will be removed in v0.20. Please import from the top-level `great_expectations` module.",
+        DeprecationWarning,
+    )
+
+    return context_factory(*args, **kwargs)
```

Comment:
Is this the best way to do this?

Reply:
I don't want users poking around util files

Reply:
Yeah, this makes sense.